admin/openwisp-monitoring/Makefile

```diff
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openwisp-monitoring
 PKG_VERSION:=0.2.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
@feckert I have bumped the PKG_RELEASE. It is part of the second commit. Is that okay?

Reply:
Fine with me